assets/csidriveroperators/shared-resource/05_clusterrole.yaml

```
@@ -88,8 +88,18 @@ rules:
   resources:
   - customresourcedefinitions
   verbs:
-  - list
   - create
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  resourceNames:
+  - sharedconfigmaps.sharedresource.openshift.io
+  - sharedsecrets.sharedresource.openshift.io
+  verbs:
+  - get
+  - update
+  - list
   - watch
   - delete
```

Apologies, forgot to add `patch` to the list of verbs here. I tend to put the verbs in Read -> Create -> Update -> Delete ordering:

```suggestion
  verbs:
  - get
  - list
  - watch
  - update
  - patch
  - delete
```